src/Xamarin.Android.Build.Tasks/Tasks/CreateLibraryResourceArchive.cs

```diff
@@ -101,6 +103,27 @@ public override bool Execute ()
 			}
 			return true;
 		}
+
+		void FixApplicationIdInManifests (string directory)
+		{
+			var broken = new List<string> ();
+			string goodFile = null;
+			foreach (var file in Directory.EnumerateFiles (directory, "AndroidManifest.xml", SearchOption.AllDirectories)) {
+				string text = File.ReadAllText (file);
+				if (text.Contains (("dollar_openBracket_applicationId_closeBracket"))) {
+					broken.Add (file);
+				} else {
+					goodFile = file;
+				}
+			}
+			if (!string.IsNullOrEmpty (goodFile)) {
+				foreach (var file in broken) {
+					DateTime lastWriteTime = File.GetLastWriteTimeUtc (file);
+					File.Copy (goodFile, file, overwrite: true);
```

Comment:
I think we decided to only use the one from the root directory not the aapt directory.